examples/combobox/aria1.1pattern/listbox-combo.html

```diff
@@ -284,7 +284,7 @@ <h3 id="kbd_label_listbox">Listbox Popup</h3>
               </ul>
             </td>
           </tr>
-          <tr>
+          <tr data-test-id="key-up-arrow">
```

Comment:
This is same ID as line 199, but won't tests be different?

Reply:
Changing this to: `listbox-key-up-arrow` and the other to `textbox-key-up-arrow`